### ISSUE TITLE
Good Spawn Location Thread

### ISSUE THREAD

**Issue by pokemonbotacc** [finndev/PokeBuddy#197]
Disney land in Japan - MagiKarp, Magnemite, Psyduck, voltorb
Venice Beach, CA - Sand Shrew, geodude, ryhorn
Central Park NY - Got a decent amount of Charmanders more than other locations i've tried.

Im looking for a good dragonair spawn, lickitung, and squirtle.

**Comment by IamDeric** [comment]
Form mason center, CA - pichachu, squirtle, dratini, dragonair
pier 39, CA - squirtle, dratini, magikarp, eevee, psyduck, tentacool

**Comment by semplex111** [comment]
can u guys post the coordinations as well?

**Comment by apollon1954** [comment]
Try Brisbane and south brisbane Australia. Jinxes and many pokestops

**Comment by aniketk11** [comment]
Pune, India: Geodude, Machop, Ekans, Sandshrew

**Comment by MikeTheory** [comment]
Saddlebrook county park in NJ is charmander spawn heaven! Tons of electric pokes in central park NY as well!

**Comment by 404unknown** [comment]
**[Pier39]** _Lat:_ 37.808966 _Long:_ -122.410153  
Got a lot of **Dratini**  and **Magikarp** there, already evolved 3 **Dragonite** and countless amount of **Gyarados** in ~2days

**Comment by ShupengHu** [comment]
When the Pokevision was available, I found that the spawn of Dratini and Dragonari was along the north and east beachline of San Francisco.  But without PokeVision and GPS spoofing, it is very hard to catch them. So now with PokeBuddy which move range seems to be a rectangle, I recommend the coordination
37.810902, -122.410820
where is a bit higher than the above one with 3.6km/h(1m/s) moving speed becasue trainer will get out of this area if you move too fast. This aims to catch every Draniti spawned in this area but without maximum exps as other places.

**Comment by PDeception** [comment]
New York, Central Park
Lat: 40.771133 
Long: -73.974187

Santa Monica Pier
Lat: 34.009606
Long: -118.496812

San Francisco
Lat: 37.7510669
Long: -122.49731910000003

**Comment by Ronixl** [comment]
Do you put walking speed to 0 when you do this so you stay still and capture pokemon at the location? Is that how it works?

**Comment by SirButtFux** [comment]
>Do you put walking speed to 0 when you do this so you stay still and capture pokemon at the location? Is that how it works?

No, since there are random spawns, you will barely get pokemon sitting idly by.

**Comment by futi666** [comment]
australia opera house is very nice :D a lot of pokestops and good pokemon
Lat: -33.856784
Long:151.215297

**Comment by MikeTheory** [comment]
Anyone know any good mapping sites that are live like pokevision? Looking for ones i can use on the PC! Thanks!
